pkg/graph/resolvers/plan_discussion.go

```diff
@@ -77,31 +77,76 @@ func CreatePlanDiscussion(
 
 		userRole := discussion.UserRole.Humanize(models.ValueOrEmpty(discussion.UserRoleDescription))
 
-		// Send email to MINT Dev Team
+		// Send email to MINT Dev Team and those subscribed to discussion created emails
 		go func() {
```

Comment:
I noted elsewhere, but I think there should probably be some refactoring around fetching preferences so that it doesn't go to the db as much.